Sources/KituraContracts/ClousureAliases.swift

```diff
@@ -18,29 +18,53 @@
 
 /**
 The `ResultClosure` is used by other `Codable` aliases when responding with only a `RequestError` is needed
```

Comment:
You need to go through this file and remove the "In this example,... " wording from the start of the description of each alias, as these are supposed to define the aliases and aren't examples.

For example "In this example, the `CodableResultClosure` is used by other `Codable` aliases..." should be "The `CodableResultClosure` is used by other `Codable` aliases..."

Please can you do this for all the aliases in this file.